src/test/java/com/snowplowanalytics/snowplow/tracker/payload/TrackerPayloadTest.java

```diff
@@ -23,6 +24,27 @@
 
 public class TrackerPayloadTest {
 
+    @Test
+    public void testGetEventId() {
+        TrackerPayload payload = new TrackerPayload();
+        // this throws an exception if it's not a valid UUID string
+        UUID.fromString(payload.getEventId());
+
+        assertTrue(payload.getMap().containsKey("eid"));
+        assertEquals(payload.getEventId(), payload.getMap().get("eid"));
+    }
+
+    @Test
+    public void testGetDeviceCreatedTimestamp() {
+        long currentTime = System.currentTimeMillis();
+        TrackerPayload payload = new TrackerPayload();
+        long timeDifference = payload.getDeviceCreatedTimestamp() - currentTime;
+        assertTrue(timeDifference < 1000);
```

Comment:
I guess it could be even less than 1000.
Not a big thing btw.

Reply:
actually 100 was too low!